Code/Framework/AzFramework/Platform/Windows/AzFramework/Device/DeviceAttributesCommon_Win.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/PlatformIncl.h>
+#include <AzCore/std/string/conversions.h>
+#include <AzFramework/Device/DeviceAttributes.h>
+
+namespace AzFramework
+{
+    DeviceAttributeDeviceModel::DeviceAttributeDeviceModel()
+    {
+        HKEY hKey = nullptr;
+        LSTATUS returnCode = ::RegOpenKeyEx(HKEY_LOCAL_MACHINE, TEXT("HARDWARE\\DESCRIPTION\\System\\BIOS"), 0, KEY_READ, &hKey);
+        if (returnCode == ERROR_SUCCESS)
+        {
+            wchar_t buf[255] = { 0 };
+            DWORD dwBufSize = sizeof(buf);
+            DWORD type = REG_SZ;
+            if (::RegQueryValueEx(hKey, TEXT("SystemProductName"), NULL, &type, reinterpret_cast<LPBYTE>(buf), &dwBufSize) == ERROR_SUCCESS)
+            {
+                AZStd::to_string(m_value, buf);
+            }
+            ::RegCloseKey(hKey);
+        }
+    }
+
+    DeviceAttributeRAM::DeviceAttributeRAM()
+    {
+        m_value = 0;

```

Comment:
repeat: No need to initalize the m_value member, if it is set to 0 in the member initialier list
```suggestion

```